### PR TITLE
chore(docs): Update additional info section ISP-117

### PR DESCRIPTION
**Pull request by anupriya0703**
FAQ has been added as a new category with Genera FAQ and cStor FAQs as subcategories. 

Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: docs/main/faqs/cStor.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
+---
+id: cStorFAQ
+title: cStor FAQs
+keywords: 
+ - OpenEBS FAQ
+ - FAQs
+description: The FAQ section about OpenEBS helps to address common concerns, questions, and objections that users have about OpenEBS.
```
[the remark this answers is not in the document]
Comment (satyapriyamishra222) [reply docs/main/faqs/cStor.md:7]: This document contains frequently asked questions about OpenEBS.

File: docs/main/faqs/general.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
+---
+id: general
+title: OpenEBS FAQs
+keywords: 
+ - OpenEBS FAQ
```
Comment (satyapriyamishra222) [inline docs/main/faqs/general.md:5]: Please add 'General FAQ about OpenEBS' in the keywords.

File: docs/main/faqs/general.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+
+[Go to top](#top)
+
+### How does OpenEBS provide high availability for stateful workloads?
+
+An OpenEBS Jiva volume is a controller deployed during OpenEBS installation. Volume replicas are defined by the parameter that you set. The controller is an iSCSI target while the replicas play the role of a disk. The controller exposes the iSCSI target while the actual data is written. The controller and each replica run inside a dedicated container. An OpenEBS Jiva volume controller exists as a single instance, but there can be multiple instances of OpenEBS Jiva volume replicas. Persistent data is synchronized between replicas. OpenEBS Jiva volume high availability is based on various scenarios as explained in the following sections. 
+
+**Note:** Each replica is scheduled in a unique K8s node, and a K8s node never has two replicas of one OpenEBS volume.
+
+### What are the recommended iscsi timeout settings on the host?
```
Comment (satyapriyamishra222) [inline docs/main/faqs/general.md:176]: is iscsi an acronym ? Then it would be better to uppercase it.

File: docs/main/faqs/general.md [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+
+There are cases when application pod and OpenEBS cStor target pod are running on different nodes. In such cases, there may be chances that application can go to read only when K8s takes around 5 mins to re-schedule OpenEBS target pod to a new Node. To avoid such scenarios, default iscsi timeout values can be configured to the recommended one. 
+
+#### Configure the iscsi timeout value
```
Comment (satyapriyamishra222) [inline docs/main/faqs/general.md:180]: Same here .

File: docs/main/faqs/general.md [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+The following explains the configuration change for 2 different scenarios.
+
+1. For New iSCSI sessions
+2. For those sessions already logged in to iSCSI target.
+
+**For New iSCSI sessions**:
```
Comment (satyapriyamishra222) [inline docs/main/faqs/general.md:186]: New => new

File: docs/main/faqs/general.md (hunk 3 above)
Comment (anupriya0703) [reply docs/main/faqs/general.md:176]: Yes, it is. Changing it.

**Review by IsAmrish — APPROVED** [review]
LGTM

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by rakeshPRaghu — APPROVED** [review]
LGTM

**Pull request merged**